tests/php/Forms/GridField/GridFieldTest.php

```diff
@@ -299,7 +296,8 @@ public function testGetColumnMetadataBadArguments()
     public function testHandleActionBadArgument()
     {
         $this->expectException(\InvalidArgumentException::class);
-        $obj = new GridField('testfield', 'testfield');
+        $list = ArrayList::create();
+        $obj = new GridField('testfield', 'testfield', $list);
```

Comment:
Needed to avoid `LogicException` now that `setThrowExceptionOnBadDataType()` is removed from various components.